scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/util/DataValidationHelper.java

```diff
@@ -55,32 +56,37 @@ public static boolean isValidTimeZone(String timeZoneId) {
 	}
 
 	/**
-	 * Checks if the specified time in milli seconds is after now (current time).
-	 * 
-	 * @param timeInMillis
+	 * Checks if the specified date time is after now (current time).
+	 * @param dateTime
 	 * @param timeZone
-	 * 
 	 * @return
 	 */
-	public static boolean isLaterThanNow(Long timeInMillis, TimeZone timeZone) {
-		if (timeInMillis != null && timeZone != null) {
-			Calendar calendar = Calendar.getInstance(timeZone);
-			Long nowInMillis = calendar.getTimeInMillis();
-
-			return timeInMillis > nowInMillis;
+	public static boolean isLaterThanNow(Date dateTime, TimeZone timeZone) {
```

Comment:
Just a general comment for future reference.

Your naming is a bit mixed. Here you use isLaterThanNow and below you use isAfter. Shouldn't this be isAfterNow?

Why don't you just do: return calToCompare.after(calNow); instead of the if block?